File: conversions/django/object.py
from django.db import models
from .abstract_element_model import AbstractElementModel

class Object(AbstractElementModel):

    # Form
    aesthetics = models.TextField(blank=True, null=True)
    weight = models.PositiveIntegerField(blank=True, null=True)
    amount = models.PositiveIntegerField(blank=True, null=True)
    parent_object = models.ForeignKey("Object", on_delete=models.SET_NULL, blank=True, null=True, related_name="object_parent_object")
    technology = models.ManyToManyField("Construct", blank=True, related_name="object_technology")

    # Function
    utility = models.TextField(blank=True, null=True)
    effects = models.ManyToManyField("Phenomenon", blank=True, related_name="object_effects")
    enables = models.ManyToManyField("Ability", blank=True, related_name="object_enables")
    consumes = models.ManyToManyField("Construct", blank=True, related_name="object_consumes")

    # World
    origins = models.TextField(blank=True, null=True)
    location = models.ForeignKey("Location", on_delete=models.SET_NULL, blank=True, null=True, related_name="object_location")

    # Games
    craftsmanship = models.TextField(blank=True, null=True)
    requirements = models.TextField(blank=True, null=True)
    durability = models.TextField(blank=True, null=True)
    value = models.PositiveIntegerField(blank=True, null=True)
    damage = models.PositiveIntegerField(blank=True, null=True)
    armor = models.PositiveIntegerField(blank=True, null=True)
    rarity = models.TextField(blank=True, null=True)
    language = models.ForeignKey("Language", on_delete=models.SET_NULL, blank=True, null=True, related_name="object_language")
    requires = models.ManyToManyField("Trait", blank=True, related_name="object_requires")

    def __str__(self):
        return self.name

File: conversions/csharp_unity_json_properties/Object.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using Utils;

[System.Serializable]
public class Object : Element
{
    [JsonProperty("aesthetics"), TextAttribute("")]
    public string aesthetics;
    [JsonProperty("weight"), Integer(0)]
    public int weight;
    [JsonProperty("amount"), Integer(0)]
    public int amount;
    [JsonProperty("parent_object"), ReferenceAttribute(typeof(Object))]
    public string parentObject;
    [JsonProperty("technology"), ReferenceAttribute(typeof(Construct), true)]
    public string technology;
    [JsonProperty("utility"), TextAttribute("")]
    public string utility;
    [JsonProperty("effects"), ReferenceAttribute(typeof(Phenomenon), true)]
    public string effects;
    [JsonProperty("enables"), ReferenceAttribute(typeof(Ability), true)]
    public string enables;
    [JsonProperty("consumes"), ReferenceAttribute(typeof(Construct), true)]
    public string consumes;
    [JsonProperty("origins"), TextAttribute("")]
    public string origins;
    [JsonProperty("location"), ReferenceAttribute(typeof(Location))]
    public string location;
    [JsonProperty("craftsmanship"), TextAttribute("")]
    public string craftsmanship;
    [JsonProperty("requirements"), TextAttribute("")]
    public string requirements;
    [JsonProperty("durability"), TextAttribute("")]
    public string durability;
    [JsonProperty("value"), Integer(0)]
    public int value;
    [JsonProperty("damage"), Integer(0)]
    public int damage;
    [JsonProperty("armor"), Integer(0)]
    public int armor;
    [JsonProperty("rarity"), TextAttribute("")]
    public string rarity;
    [JsonProperty("language"), ReferenceAttribute(typeof(Language))]
    public string language;
    [JsonProperty("requires"), ReferenceAttribute(typeof(Trait), true)]
    public string requires;
}
